specificatie/kadaster-niet-natuurlijk-personen.yaml

```diff
@@ -53,7 +53,7 @@ paths:
     get:
       operationId: GetKadasterNietNatuurlijkPersonen
       description: |
-                    Het zoeken van bij het kadaster geregistreerde niet natuurlijke personen die al dan niet in het handelsregister (HR) zijn ingeschreven. Kadasternietnatuurlijkpersonen worden niet geactualiseerd.
+                    Zoek bij het kadaster geregistreerde niet natuurlijke personen die niet in het Handelsregister (HR) zijn ingeschreven, of wel zijn ingeschreven maar niet gekoppeld bij het inschrijven van de akte. Kadasternatuurlijkpersonen worden niet geactualiseerd.
```

Comment:
Elke zin in een eigen regel? Dit gebeurt soms wel en soms niet. Is wel makkelijker bij het lezen. Dan hoeft je niet heen en weer te scrollen